src/Components/Aspire.Microsoft.Azure.Cosmos/PublicAPI.Unshipped.txt

```diff
@@ -10,5 +10,5 @@ Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.DisableTracing.get
 Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.DisableTracing.set -> void
 Aspire.Microsoft.Azure.Cosmos.MicrosoftAzureCosmosDBSettings.MicrosoftAzureCosmosDBSettings() -> void
 Microsoft.Extensions.Hosting.AspireMicrosoftAzureCosmosDBExtensions
```

Comment:
```suggestion
Microsoft.Extensions.Hosting.AspireMicrosoftAzureCosmosExtensions
```

Reply:
Thanks! This is now covered with the latest commit.